### PR TITLE
[CDAP-8282] add View Events Fast Action

### PR DESCRIPTION
**Pull request by elfenheart**
JIRA: https://issues.cask.co/browse/CDAP-8282

### CODE REVIEW

**Comment by tonybach** [comment]
At first, I tried setting the start time to be later than the end time, and the error was displayed correctly. However, the error doesn't go away even when I have changed the time range back correctly.

Before:
screen shot 2017-02-14 at 6 03 34 pm

After:
screen shot 2017-02-14 at 6 03 46 pm

**Comment by ajainarayanan** [comment]
@elfenheart Noticed couple of things,
1. The size of the modal seems to be smaller. Based on the mock in the JIRA I think we could use the size of explore modal
2. The number textbox for #of events can we have the min as 1. It right now goes to negative. Could save a backend call.

File: cdap-ui/app/cdap/components/FastAction/ViewEventsAction/ViewEventsModal.js [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+
+    return (
+      <Modal
+        isOpen={true}
+        toggle={this.props.onClose}
+        className="confirmation-modal view-events-modal"
+        size="lg"
+        backdrop='static'
+      >
+        <ModalHeader>
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/FastAction/ViewEventsAction/ViewEventsModal.js]: nit: We might need a `clearfix` class here.

**Review by ajainarayanan — COMMENTED** [review]
Minor comments. Otherwise 👍 LGTM

**Comment by tonybach** [comment]
👍  LGTM

**Comment by elfenheart** [comment]
Build: http://builds.cask.co/browse/CDAP-DRC5715-6

**Pull request merged**